### DISCUSSION TITLE
Document AI expense parser does not work

### DISCUSSION THREAD

**Discussion by aimagination-dev** [AntlerVC/firetable#1344] [Support / Q&A] [answered]
I went through the complete setup procedure for the template "Document AI expense parser" but after adding an image the nothing happens on the row. Any idea what might be wrong?

**Comment by fraverdev** [comment]
I have the same problem, have you solved it already ?

**Reply by fraverdev** [reply]
If you use the template Document AI, something goes wrong in firestore. 

this is what i have in my firestore:
![Screenshot 2023-07-20 at 14 44 59](https://github.com/rowyio/rowy/assets/120483187/90dc68a6-f1fe-4897-a18a-bb2ccebacc1d)
<img width="833" alt="Screenshot 2023-07-20 at 14 45 06" src="https://github.com/rowyio/rowy/assets/120483187/c8441613-59b6-4e5a-a33d-3e15c89fc7e4">

and this is what i should have, can someone provide me with the parsedData fields?

**Reply by chrisat3Seven** [reply]
Did you ever resolve this. This is the exact problem I have. I've deleted my rowy app and redeployed it numerous times with the same result. I always get an error on the first attempt at deploying so wondering if it's related.

**Reply by fraverdev** [reply]
Oh that is a long time ago. When you setup the server location on your google parser, I believe that EU doesn't work. Then I just changed my firebase location to US and set the google parser to US. Idk why the EU server diddn't work. Eventually I created my own api with the google parser.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @aimagination-dev, @fraverdev, could you check your [logs](https://docs.rowy.io/cloud-logs) for any indicated error? Also, you could try using some logging statements to debug your functions.
